tools/tessolarcharge.py

```diff
@@ -150,6 +149,7 @@ async def TesSolarCharge() :
             print('Use browser to login. Page Not Found will be shown at success.')
             print('Open this URL: ' + tesla.authorization_url())
             tesla.fetch_token(authorization_response=input('Enter URL after authentication: '))
+		global vehicles			   
```

Comment:
Check this indent as well.  Seems like this would throw and exception.  I recommend running `pylint` to ensure there isn't anything off.

Reply:
yes, thanks for catching and your patience! Adding pylint to my workflow!